app/marketing/management/commands/expiration_start_work.py

```diff
@@ -99,28 +99,45 @@ def handle(self, *args, **options):
                             print(" - no actions")
                         else:
                             # example format: 2018-01-26T17:56:31Z'
-                            action_times = [
-                                datetime.strptime(action['created_at'], '%Y-%m-%dT%H:%M:%SZ') for action in actions
-                                if action.get('created_at')
-                            ]
-                            last_action_by_user = max(action_times).replace(tzinfo=pytz.UTC)
-
-                            # if user hasn't commented since they expressed interest, handled this condition
-                            # per https://github.com/gitcoinco/web/issues/462#issuecomment-368384384
-                            if last_action_by_user.replace() < interest_day_0:
-                                last_action_by_user = interest_day_0
-
-                            # some small calcs
-                            snooze_time = timezone.timedelta(days=bounty.snooze_warnings_for_days)
-                            delta_now_vs_last_action = timezone.now() - snooze_time - last_action_by_user
-                            last_heard_from_user_days = delta_now_vs_last_action.days
-
-                            # decide action params
-                            should_warn_user = last_heard_from_user_days >= num_days_back_to_warn
-                            should_delete_interest = last_heard_from_user_days >= num_days_back_to_delete_interest
-                            should_ignore = last_heard_from_user_days >= num_days_back_to_ignore_bc_mods_got_it
-
-                            print(f"- its been {last_heard_from_user_days} days since we heard from the user")
+                            action_times = []
```

Comment:
whyd you remove the `datetime.strptime(action['created_at'], '%Y-%m-%dT%H:%M:%SZ') ` from this list?

Reply:
ah nvm, isee it was moved below

Reply:
This was the first bug, see variable assignment. 😄

Reply:
Ah, sry that was no bug here. I had my mind on the wrong file